source/grid/manifold_lib.cc

```diff
@@ -32,11 +32,13 @@ SphericalManifold<dim,spacedim>::SphericalManifold(const Point<spacedim> center)
 
 
 template <int dim, int spacedim>
-Point<spacedim>
+Tensor<1,spacedim>
 SphericalManifold<dim,spacedim>::get_periodicity()
 {
-  Point<spacedim> periodicity;
-  periodicity[spacedim-1] = 2*numbers::PI; // theta and phi period.
+  Tensor<1,spacedim> periodicity;
+  // In two dimensions, theta is periodic. In three dimensions the variable
+  // that is periodic is phi. We have no periodicity in theta in 3d.
+  periodicity[spacedim-1] = 2*numbers::PI;
```

Comment:
Hm, it is quite possible that it doesn't matter in practice. But there *is* some periodicity in two directions -- it just happens to be a rather complicated kind of periodicity in the polar angle. Maybe update the comment to say so?